### PR TITLE
Added funcionality to export to CSV. Supports filtered results.

### PR DESCRIPTION
**Pull request by MarcoLPR**
I rearranged the structure of the button, making it part of the Toolbar component instead of being an external one, also now supports export filtered results.

### CODE REVIEW

**Comment by ArCiGo** [comment]
Conflicts D:

File: src/Grid/Grid.js [hunk 1]
```diff
@@ -135,17 +134,83 @@ class Grid extends React.Component {
         popup.document.close();
       });
   }
+  
+  exportTable = filtered => {
```
Comment (geoperez) [inline src/Grid/Grid.js]: @kadosh do we need this method in the same class?

Comment (kadosh) [reply src/Grid/Grid.js]: Maybe we will move that one and printTable as well to an API class (something similar to an angular service).

Comment (geoperez) [reply src/Grid/Grid.js]: Yeah, let's keep this class for now, and refactor it with typescript!

**Pull request merged**